### PR TITLE
[MIG][14.0] Analytic Brand

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

File: analytic_brand/views/res_brand.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <field name="model">res.brand</field>
         <field name="inherit_id" ref="brand.res_brand_form_view" />
         <field name="arch" type="xml">
-            <xpath expr="//sheet" position="inside">
+            <xpath expr="//form" position="inside">
```
Comment (sbejaoui) [inline analytic_brand/views/res_brand.xml:9]: why this change?

Comment (AmmarOfficewalaSerpentCS) [reply analytic_brand/views/res_brand.xml:9]: sheet tag does not use in a parent that's why I use a form instead of sheet.

**Review by sbejaoui — APPROVED** [review]

**Comment by sbejaoui** [comment]
please rebase on 14.0

**Comment by AmmarOfficewalaSerpentCS** [comment]
Okay, 
Let me do
